src/common/health.rs

```diff
@@ -181,6 +181,9 @@ impl Task {
 
         let peer_address_by_id = self.consensus_state.peer_address_by_id();
 
+        // Endpoint we check is not available before Qdrant 1.7.0, allow checks on nodes with earlier versions to fail by counting them here
```

Comment:
Should we have a clear TODO to remove this trick in Qdrant 1.8.0?

Reply:
We can do, but it doesn't hurt to keep it here for a bit longer.